modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1031,7 +1031,8 @@ class Edit_User extends \NDB_Form
                 || $values['examiner_pending'] !== '')
             ) {
                 $errors['examiner_sites'] = "Please select at least one examiner 
-                site.";
+                site or reset the 'Examiner status' fields below (i.e. 
+                'Radiologist' and 'Pending Approval') to nothing.";
```

Comment:
could we say something like "clear the 'Examiner status' fields below ..."? to replace "to nothing"